hackathon: prettify

thelostone-mc opened the pull request:
##### Description

- update the banner to make it a lil easier on the eyes
- update the title + subheading to match the rest of the site
- add a bit of animation
- give more breathing space to the section where bounties are displayed & sidebar search
- update STATUS -> ORGS in the sidebar search
- removed unused styles/code
- link on bounty detail page for a hackathon bounty is updated to point to the hackathon page as opposed to the explorer [AS SHOWN HERE](https://user-images.githubusercontent.com/5358146/56428443-fc7eb400-62dc-11e9-9d6f-f8b0bf051920.gif)




Recording : https://gitcoincore.slack.com/archives/CGR0V2TRV/p1555590611002900

alexvotofuture commented:
Hey @thelostone-mc, thanks for working to make the prize explorer even better!

Some feedback on the changes:
- Banner should be purple, similar to the [Ethereal Summit branding](https://etherealsummit.com/)
- Change "orgs" to "Sponsors"
- Can we eliminate the "network" filter on the sidebar entirely? We only want the prizes deployed to main net anyway

Great stuff. Will post some additional bug fix issues separately. Thank you!

thelostone-mc commented:
@alexvotofuture 

- Making the banner just blue seemed a lil dull ! 
I updated it to the red & blue from Ethereal Summit to create a gradient effect as[ SHOWN HERE](https://user-images.githubusercontent.com/5358146/56426816-2b922700-62d7-11e9-8a7a-50ccaa01ee4b.png)
Would that be alright ?

- updated it to sponsors
- updated it to hide network on live. (will still be view in dev env to make testing easier)
